File: Structural/Bridge/Bridge_Java/TVDevice.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Bridge_Java;

/**
 *
 * @author shantanu1395
 */
public class TVDevice extends GenericDevice{

    public TVDevice(int newdeviceState,int newMaxSetting) {
        deviceState=newdeviceState;
        maxSetting=newMaxSetting;
    }

    @Override
    public void buttonFivePressed() {
        System.out.println("Channel Down");
        deviceState--;
    }

    @Override
    public void buttonSixPressed() {
        System.out.println("Channel Up");
        deviceState++;
    }
    
}

File: Structural/Bridge/Bridge_py/TVDevice.py
from GenericDevice import *

class TVDevice(GenericDevice):
    def __init__(self,deviceState,maxSetting):
        self.deviceState=deviceState
        self.maxSetting=maxSetting

    def buttonFivePressed(self):
        print("Channel Down")
        self.deviceState-=1

    def buttonSixPressed(self):
        print("Channel Up")
        self.deviceState+=1